test/send.c

```diff
@@ -34,19 +34,72 @@ static void iface_teardown(void)
 	iface = 0;
 }
 
-START_TEST(test_add_ra_header)
+START_TEST(test_add_ra_header_cease_adv0)
 {
 
 	struct safe_buffer sb = SAFE_BUFFER_INIT;
-	add_ra_header(&sb, &iface->ra_header_info, iface->state_info.cease_adv);
+	add_ra_header(&sb, &iface->ra_header_info, 0);
+
+#ifdef PRINT_SAFE_BUFFER
+	char buf[4096];
+	snprint_safe_buffer(buf, 4096, &sb);
+	ck_assert_msg(0, "\n// ra_header_info->AdvDefaultLifetime = %x\n%s", iface->ra_header_info.AdvDefaultLifetime, &buf);
+#else
+	// Lifetime should be -1/ffff in this case, because we have not set it to anything else.
+	// interface.c:40:iface->ra_header_info.AdvDefaultLifetime = -1;
+	unsigned char expected[] = {
+		// nd_ra_type
+		// nd_ra_code
+	    0x86, 0x00,
```

Comment:
Looks like we've got mixed tabs and spaces on this line. Let's stick to tabs since they are already being used throughout this code.

Reply:
fixed